sensor/kubernetes/fake/fake.go

```diff
@@ -61,12 +65,12 @@ func (c *clientSetImpl) Kubernetes() kubernetes.Interface {
 
 // OpenshiftApps returns nil for the openshift client for config
```

Comment:
This comment is not accurate anymore, now that it returns an actual client, right?

Reply:
Good catch. It should be fixed now.